pkgs/data/icons/bibata-cursors/default.nix

```diff
@@ -1,51 +1,54 @@
-{ lib, stdenvNoCC, fetchFromGitHub, gnome-themes-extra, inkscape, xcursorgen, python3 }:
-
-let
-  py = python3.withPackages(ps: [ ps.pillow ]);
-in stdenvNoCC.mkDerivation rec {
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchurl
+, clickgen
+, unzip
+}:
+
+stdenv.mkDerivation rec {
   pname = "bibata-cursors";
-  version = "0.4.2";
+  version = "1.1.2";
 
   src = fetchFromGitHub {
-    owner = "KaizIqbal";
+    owner = "ful1e5";
```

Comment:
Seems like the user changed its nickname, because https://github.com/KaizIqbal/Bibata_Cursor points to https://github.com/ful1e5/Bibata_Cursor.